app/build.gradle

```diff
@@ -491,4 +494,43 @@ afterEvaluate {
             }
         }
     }
+
+    // Format test output. Ported from AC #2401
+    tasks.matching {it instanceof Test}.all {
+        testLogging.events = []
+
+        def out = services.get(StyledTextOutputFactory).create("an-ouput")
```

Comment:
Does "an-ouput" stand for anything? It looks like that parameter is for "logging category", perhaps "tests" would be more adequate?